diff --git a/node_modules/@serverless-stack/cli/bin/scripts.js b/node_modules/@serverless-stack/cli/bin/scripts.js
index c376d22..7b03987 100755
--- a/node_modules/@serverless-stack/cli/bin/scripts.js
+++ b/node_modules/@serverless-stack/cli/bin/scripts.js
@@ -18,6 +18,7 @@ const fs = require("fs-extra");
 const yargs = require("yargs");
 const chalk = require("chalk");
 const spawn = require("cross-spawn");
+const fp = require('lodash/fp');
 const { logger, initializeLogger } = require("@serverless-stack/core");
 
 const packageJson = require("../package.json");
@@ -358,7 +359,17 @@ switch (script) {
 
     // Prepare app
     prepareCdk(argv, cliInfo, config)
-      .then(() => internals[script](argv, config, cliInfo))
+      .then(() => {
+      const res = internals[script](argv, config, cliInfo);
+       res.then(
+         fp.flow(
+           fp.map('status'),
+           fp.indexOf('failed'),
+           fp.tap(found => process.exit(found >= 0))
+         )
+       );
+       return res;
+      })
       .catch((e) => exitWithMessage(e.message));
 
     break;
